src/jit/compiler.cpp

```diff
@@ -6607,6 +6607,33 @@ void Compiler::GetStructTypeOffset(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSIN
         *type1 = GetEightByteType(structDesc, 1);
     }
 }
+
+//------------------------------------------------------------------------------------------------------
+// GetStructTypeOffset: Gets the type, size and offset of the eightbytes of a struct for System V systems.
+//
+// Arguments:
+//    'typeHnd'    -  type handle
+//    'type0'      -  out param; returns the type of the first eightbyte.
+//    'type1'      -  out param; returns the type of the second eightbyte.
+//    'offset0'    -  out param; returns the offset of the first eightbyte.
+//    'offset1'    -  out param; returns the offset of the second eightbyte.
+//
+void Compiler::GetStructTypeOffset(const CORINFO_CLASS_HANDLE& typeHnd,
+                                   var_types*                  type0,
+                                   var_types*                  type1,
+                                   unsigned __int8*            offset0,
+                                   unsigned __int8*            offset1)
+{
+    SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR structDesc;
+    eeGetSystemVAmd64PassStructInRegisterDescriptor(typeHnd, &structDesc);
+    assert(structDesc.passedInRegisters);
+    *offset0 = 0;
+    *offset1 = 0;
+    *type0   = TYP_UNKNOWN;
+    *type1   = TYP_UNKNOWN;
```

Comment:
These are guaranteed to be set by the GetStructTypeOffset() call, so there's no need to initialize them, IMO.